src/styles/boxes/stars__star.css

```diff
@@ -0,0 +1,23 @@
+.stars--0 .stars__star:nth-child(-n + 0) {
+  background-image: url(../../images/star-active.svg);
+}
+
+.stars--1 .stars__star:nth-child(-n + 1) {
+  background-image: url(../../images/star-active.svg);
+}
+
+.stars--2 .stars__star:nth-child(-n + 2) {
+  background-image: url(../../images/star-active.svg);
+}
+
+.stars--3 .stars__star:nth-child(-n + 3) {
+  background-image: url(../../images/star-active.svg);
+}
+
+.stars--4 .stars__star:nth-child(-n + 4) {
+  background-image: url(../../images/star-active.svg);
+}
+
+.stars--5 .stars__star:nth-child(-n + 5) {
+  background-image: url(../../images/star-active.svg);
+}
```

Comment:
No need to create a separate file as stars__star is an element of stars block. You can also combine selectors using comma

.selector1,
.selector2 {
  identical styles here...
}